app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    if params[:cont_id]
```

Comment:
A more RESTful approach to this would be:

Have your form re-route to a different controller action. The most RESTful option could be in a joins controller like the ContestantProjectsController you made above. You could route to a `create` action there to associate a contestant with a project. Then from that action, you could redirect back to the project show page. 

Right now your project show action is doing a bit too much. To follow RESTfulness and SRP, you can break up this logic